stdlib/array.ncl

```diff
@@ -21,30 +21,50 @@
        else
          %blame% (%tag% "not a array" label),
 
-    head : forall a. Array a -> a
+    first : forall a. Array a -> a
       | doc m%"
-        Results in the head (first element) of the given array.
+        Results in the first element of the given array.
```

Comment:
Nitpick: I think "Returns..." is a little more direct than "Results in..." though both are fine.

Reply:
Yep, totally fair.